module/src/components/listBox/listBox.basic.scss

```diff
@@ -25,6 +20,7 @@
   display: flex;
   align-items: center;
   cursor: pointer;
+  min-width: 160px;
```

Comment:
Does this specific value need to be documented? Should it come from a variable?

Reply:
in a variable now